presto-main/src/main/java/com/facebook/presto/sql/planner/PlanPrinter.java

```diff
@@ -1213,7 +1214,7 @@ private void printAssignments(Assignments assignments, int indent)
 
         private String formatOutputs(Iterable<Symbol> symbols)
         {
-            return Joiner.on(", ").join(Iterables.transform(symbols, input -> input + ":" + types.get(input).getDisplayName()));
+            return Joiner.on(", ").join(Iterables.transform(symbols, input -> input + ":" + types.getOrDefault(input, UNKNOWN).getDisplayName()));
```

Comment:
This happened when running a test that was not well-defined yet. Nevertheless, PlanPrinter should be robust enough to cope with ill-defined trees. If there is a problem with missing types in production code, it'll get caught by the code using them anyway.